examples/validate-fee/sample-ui/.eslintignore

```diff
@@ -0,0 +1,43 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Let's remove the sample-ui if it's not actually used for the example.

Reply:
Deleted sample-ui